state-chain/custom-rpc/src/lib.rs

```diff
@@ -1229,69 +1340,43 @@ where
 				return Ok(())
 			},
 		};
+		let stream = futures::stream::iter(std::iter::once(Ok(BlockUpdate {
+			block_hash: info.best_hash,
+			block_number: info.best_number,
+			data: initial.clone(),
+		})))
+		.chain(
+			self.client
+				.import_notification_stream()
+				.filter(|n| futures::future::ready(n.is_new_best))
```

Comment:
What is the meaning of `is_new_best`? Does this just mean that we ignore re-orgs?

Reply:
Yes

Reply:
Sort of, it is more intended, so we skip to the latest block (if the node discovers a whole sequence of blocks at once)